src/components/RelatiesKoppelingen/RelatiesKoppelingen.js

```diff
@@ -176,9 +188,9 @@ const RelatiesKoppelingen = ({
         }
 
         /**
-         * Function to inialize the needed data for objects that are NOT of type 'Beleidskeuze'
-         * Sets the relations to beleidskeuzes in state, and gets the active verordeningsStructure
-         * When data is set in State we set the loading state to False
+         * Function to set the intitialized data for a Beleidsobject.
```

Comment:
Why was the existing documentation replaced?

Reply:
I wanted to sum up what has been described, but I have reverted the changes.